test/lib/build.test.js

```diff
@@ -173,7 +173,6 @@ describe('Build Model', () => {
     describe('start', () => {
         let sandbox;
         let tokenGen;
-        const user = { username: 'me' };
```

Comment:
Minor, since it's just a test. Why not keep the `const user` and define it as a `screwdriver-cd` here?